system/modules/core/controllers/FrontendIndex.php

```diff
@@ -56,7 +56,9 @@ public function __construct()
 	 */
 	public function run()
 	{
+		/** @var \PageModel $objPage */
```

Comment:
Thanks a lot. I just stumbled across this, too, and I had not yet figured out :)